### PR TITLE
switched from hours to days

### PR DESCRIPTION
**Pull request by dmessing**
@skyhit could we have someone test this and redeploy?  We are moving effort hours to effort days.  It is a text change and some variable changes.  I will update the DB.

### CODE REVIEW

**Comment by skyhit** [comment]
@dmessing I am asking the winner for effort hours challenge to test this, making sure nothing is broken

File: services/client_project_entities_dao/src/java/main/com/topcoder/clients/model/Client.java [hunk 1]
```diff
@@ -112,7 +112,7 @@
      * </p>
      */
     @Column(name = "enable_effort_hours")
```
Comment (skyhit) [inline services/client_project_entities_dao/src/java/main/com/topcoder/clients/model/Client.java]: @dmessing should this be changed?

Comment (skyhit) [reply services/client_project_entities_dao/src/java/main/com/topcoder/clients/model/Client.java]: @deedee should we change the column name?

Comment (deedee) [reply services/client_project_entities_dao/src/java/main/com/topcoder/clients/model/Client.java]: @skyhit On current code, that is not required. Unless you want it

**Pull request merged**